common/Resources/ti.internal/extensions/node/internal/util/inspect.js

```diff
@@ -0,0 +1,1695 @@
+	inverse: [ 7, 27 ],
+	white: [ 37, 39 ],
+	grey: [ 90, 39 ],
+	black: [ 30, 39 ],
+	blue: [ 34, 39 ],
+	cyan: [ 36, 39 ],
+	green: [ 32, 39 ],
+	magenta: [ 35, 39 ],
+	red: [ 31, 39 ],
+	yellow: [ 33, 39 ]
+});
+
+// Don't use 'blue' not visible on cmd.exe
+inspect.styles = Object.assign(Object.create(null), {
+	special: 'cyan',
+	number: 'yellow',
+	bigint: 'yellow',
+	boolean: 'yellow',
+	undefined: 'grey',
+	null: 'bold',
+	string: 'green',
+	symbol: 'green',
+	date: 'magenta',
+	// "name": intentionally not styling
+	regexp: 'red',
+	module: 'underline'
+});
+
+function addQuotes(str, quotes) {
+	if (quotes === -1) {
+		return `"${str}"`;
+	}
+	if (quotes === -2) {
+		return `\`${str}\``;
+	}
+	return `'${str}'`;
+}
+
+const escapeFn = (str) => meta[str.charCodeAt(0)];
+
+// Escape control characters, single quotes and the backslash.
+// This is similar to JSON stringify escaping.
+function strEscape(str) {
+	let escapeTest = strEscapeSequencesRegExp;
+	let escapeReplace = strEscapeSequencesReplacer;
+	let singleQuote = 39;
+
+	// Check for double quotes. If not present, do not escape single quotes and
+	// instead wrap the text in double quotes. If double quotes exist, check for
+	// backticks. If they do not exist, use those as fallback instead of the
+	// double quotes.
+	// eslint-disable-next-line quotes
+	if (str.includes("'")) {
+		// This invalidates the charCode and therefore can not be matched for
+		// anymore.
+		if (!str.includes('"')) {
+			singleQuote = -1;
+		} else if (!str.includes('`') && !str.includes('${')) {
+			singleQuote = -2;
+		}
+		if (singleQuote !== 39) {
+			escapeTest = strEscapeSequencesRegExpSingle;
+			escapeReplace = strEscapeSequencesReplacerSingle;
+		}
+	}
+
+	// Some magic numbers that worked out fine while benchmarking with v8 6.0
+	if (str.length < 5000 && !escapeTest.test(str)) {
+		return addQuotes(str, singleQuote);
+	}
+	if (str.length > 100) {
+		str = str.replace(escapeReplace, escapeFn);
+		return addQuotes(str, singleQuote);
+	}
+
+	let result = '';
+	let last = 0;
+	const lastIndex = str.length;
+	for (let i = 0; i < lastIndex; i++) {
+		const point = str.charCodeAt(i);
+		if (point === singleQuote || point === 92 || point < 32) {
+			if (last === i) {
+				result += meta[point];
+			} else {
+				result += `${str.slice(last, i)}${meta[point]}`;
+			}
+			last = i + 1;
+		}
+	}
+
+	if (last !== lastIndex) {
+		result += str.slice(last);
+	}
+	return addQuotes(result, singleQuote);
+}
+
+function stylizeWithColor(str, styleType) {
+	const style = inspect.styles[styleType];
+	if (style !== undefined) {
+		const color = inspect.colors[style];
+		return `\u001b[${color[0]}m${str}\u001b[${color[1]}m`;
+	}
+	return str;
+}
+
+function stylizeNoColor(str) {
+	return str;
+}
+
+// Return a new empty array to push in the results of the default formatter.
+function getEmptyFormatArray() {
+	return [];
+}
+
+function getConstructorName(obj, ctx) {
+	let firstProto;
+	const tmp = obj;
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/node/internal/util/inspect.js;
  Line: 335;
-->

- :warning: common/Resources/ti.internal/extensions/node/internal/util/inspect.js line 335 – 'tmp' is assigned a value but never used. (no-unused-vars)